lib/shared/bucketing-assembly-script/assembly/types/bucketedUserConfig.ts

```diff
@@ -220,10 +220,14 @@ export class SDKVariable extends JSON.Obj {
         const stringValue = (this.type === 'String' && this.value.isString)
             ? (this.value as JSON.Str).valueOf()
             : null
-        const jsonValue = (this.type === 'JSON' && this.value.isObj)
+        let jsonValue = (this.type === 'JSON' && this.value.isObj)
             ? this.value.stringify()
             : null
 
+        if (this.type === 'JSON' && jsonValue == null && this.value.isString) {
```

Comment:
I think this was actually a bug. It seems like this was just setting the value to null for all json variables, since the value field is always a string in this context

Reply:
```suggestion
        if (jsonValue == null && this.type === 'JSON' && this.value.isString) {
```